alipay_mobile/__init__.py

```diff
@@ -150,7 +158,12 @@ def extract(self, file, existing_entries=None):
                     my_warn(
                         f"Multiple payment methods found, please confirm", lineno, row
                     )
-                    tags.add("confirmation-needed")
+                    tags.add("multiple-payment-methods") # usually means a discount and needs no special treatment
+                #     metadata["payment_method"] = method  
```

Comment:
Please don't ignore any _original_ information in the exported bills, e.g. payment method. You can rename the tag, but please keep this field in the postings/